draft-ietf-quic-qpack.md

```diff
@@ -225,9 +273,8 @@ represented as an 8-bit prefix string literal.
 ### Duplicate {#indexed-duplicate}
 
 Duplication of an existing entry in the dynamic table starts with the '000'
-three-bit pattern.  The index of the existing entry is represented as an integer
-with a 5-bit prefix. Table indices are always non-zero; a table index of zero
-MUST be treated as a decoding error.
+three-bit pattern.  The absolute index of the existing entry is represented as
```

Comment:
Why absolute?  Because it's ordered on the control stream, this can be relative to N=number of inserts.

Reply:
#1128.  You're almost certainly going to duplicate entries which have the largest possible relative index in your table.  Unless your table has wrapped multiple times, that means they have a relatively *small* absolute index.  On the flip side, the size of this will grow continuously over the lifetime of a connection while the relative index is at least bounded.

For ideal compression, this would be relative to the total number of drops (so the last item in the table is 0, etc.), but I was really hesitant to introduce a third (fourth, given #1239) way of indexing entries in the same table.

Reply:
Okay, you've convinced me.  😀